ApplicationBase.ts

```diff
@@ -532,24 +529,13 @@ class ApplicationBase {
     return (await portal.queryGroups(params)) as __esri.PortalQueryResult;
   }
 
-  private _loadItem(id: string): Promise<PortalItem> {
+  private _loadItem(id: string): IPromise<any> {
```

Comment:
can we type this instead of having any?

Reply:
Yeah I wonder why I switched to any - I'll look at it it could have been an old update that I accidentally merged back in